CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## TBD
 
+This version contains **breaking** changes, as bugsnag-unity has been updated to use the latest available versions of bugsnag-android (v4.22.2 -> v5.9.4) and bugsnag-cocoa (v5.23.5 -> v6.9.3).
+
+Please see the [upgrade guide](UPGRADING.md) for details of all the changes and instructions on how to upgrade.
+
 ### Bug fixes
 
 * Remove Bugsnag.Client from public API 
```

Comment:
Can we trim out of here the changes that are covered by the notifier uplift as they're implicit with the new paragraph above. It's probably just PRs 255 and 239 that are bug fixes?

Reply:
Removed everything except those two PRs.